chaco/ticks.py

```diff
@@ -242,6 +243,10 @@ def auto_ticks(
     An array of tick mark locations. The first and last tick entries are the
     axis end points.
     """
+    # if a NaN makes it here, we cant determine tick mark locations. Return
+    # empty list to prevent crash. Motivated by enthought/chaco#636
```

Comment:
lets link to the issue instead of the PR

```suggestion
    # empty list to prevent crash. Motivated by enthought/chaco#529
```